.pylintrc

```diff
@@ -101,7 +101,7 @@ function-rgx=[a-z_][a-z0-9_]{2,40}$
 function-name-hint=[a-z_][a-z0-9_]{2,40}$
 
 # Regular expression matching correct variable names
-variable-rgx=[a-z_][a-z0-9_]{2,30}$
+variable-rgx=[a-z_][a-z0-9_]{1,30}$
```

Comment:
Please revert:
1. This is not necessary for this PR
2. Forcing not to use too short variables is on purpose. Disable locally if you must.